libdrgn/program.h

```diff
@@ -168,6 +168,10 @@ struct drgn_program {
 		bool pgtable_l5_enabled;
 		/** PAGE_SHIFT of the kernel (derived from PAGE_SIZE). */
 		int page_shift;
+
+		/** The original vmcoreinfo data, to expose as an object */
+		char *data;
+		size_t data_size;
```

Comment:
What do you think of naming this `raw` and `raw_size`?

Reply:
That makes more sense because it's already inside `vmcoreinfo` struct, so it is clearer to understand what it contains. Data is very generic and is usually a bad variable name :)